bundles/mapping/camera-controls-3d/view/Desktop.jsx

```diff
@@ -21,63 +22,65 @@ const MapControlsContainer = styled.div`
 `;
 
 const MapControlContainer = styled.div`
-    border-radius: 50%;
-    width: 32px;
-    height: 32px;
-    cursor: pointer;
-    z-index: 15000;
     margin-bottom: 5px;
     margin-left: 5px;
-    box-shadow: ${iconShadow};
-    background-color: ${props => props.controlIsActive ? secondaryColor : darkBgColor};
-    opacity: ${props => props.disabled ? 0.3 : 1.0};
-    pointer-events: ${props => props.disabled ? 'none' : 'auto'};
-`;
-
-const MapControl = styled.div`
-    width: 32px;
-    height: 32px;
-    background-repeat: no-repeat;
-    margin: 0;
-    padding-top:3px;
-    text-align: center;
 `;
 
 const Break = styled.div`
     flex-basis: 100%;
     height: 0;
 `;
 
-export const Desktop = LocaleConsumer(({ activeMapMoveMethod, controller, getMessage }) => {
-    const mapInMobileMode = false;
-
+export const Desktop = LocaleConsumer(({ activeMapMoveMethod, controller, getMessage, styleName }) => {
     return (<MapControlsContainer>
-        <MapControlContainer controlIsActive = {activeMapMoveMethod === mapMoveMethodMove}>
-            <MapControl>
-                <MoveMapIcon mapInMobileMode={mapInMobileMode}
-                    clickHandler={() => controller.setActiveMapMoveMethod(mapMoveMethodMove)}
-                    title={getMessage('tooltip.move')} controlIsActive = {activeMapMoveMethod === mapMoveMethodMove}/>
-            </MapControl>
+        <MapControlContainer>
+            <MapModuleButton
+                onClick={() => controller.setActiveMapMoveMethod(mapMoveMethodMove)}
+                icon={<MoveMapIcon mapInMobileMode={false} />}
```

Comment:
Icons no longer handle that prop?
```suggestion
                icon={<MoveMapIcon />}
```